lib/shopifyHMAC/index.js

```diff
@@ -0,0 +1,59 @@
+'use strict'
+
+const crypto = require('crypto')
+
+const extractShopifySignature = ({ query }) => {
+  if (!query.hmac) {
+    throw new Error(
+      'Could not extract HMAC parameter: no HMAC parameter found in query.'
+    )
+  }
+  return query.hmac
+}
+
+exports.extractShopifySignature = extractShopifySignature
```

Comment:
can we do `module.exports` at the base of the file? saves repetition, and there are subtle differences between just `exports` and `module.exports`, I believe

```js
module.exports = {
  extractShopifySignature,
  constructShopifySignature
}

Reply:
Changes made